src/UserWarning.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const UserWarning: React.FC = () => (
+  <section className="section">
+    <p className="box is-size-3">
+      Please get your
+      {' '}
```

Comment:
do you need to use so many {' '} here? maybe think of other way, e.g. template literals